README.md

```diff
@@ -16,20 +16,8 @@ You can find Aiia documentation at <https://docs.aiia.eu/>
 
 For starters, you'll need to create an account at <https://portal.aiia.eu/> and generate a `ClientId` and `ClientSecret`.
 
-Then update `Aiia` configuration section in `appsettings.json`:
-
-```
-...
-  "Aiia": {
-    ...
-    "ClientId": "your-client-id",
-    "ClientSecret": "your-client-secret",
-    "WebHookSecret": "your-webhook-secret-if-configured"
-  }
-...
-```
-
-Other sections are unnecessary to modify to run the project locally.
+Then run the `initialize.sh` or `initialize.ps1` scripts to setup the project. The script will ask you the `ClientId` and `ClientSecret` you previously configured.
```

Comment:
initialize.sh/ps uses dotnet user-secrets, before we ask to download .NET core. not a big thing, given the audience (probably devs only), but maybe we can shuffle it around a bit?

Reply:
also - at least in the PS script - we start with `cd Web`, so maybe we can just add a stronger emphasis that the script needs to be run from the root of the project?